### ISSUE TITLE
NameError: free variable 'done_specs' referenced before assignment in enclosing scope

### ISSUE THREAD

**Issue by rakhimovv** [royorel/FFHQ-Aging-Dataset#6]
Hey @royorel . Thanks for sharing the solution of downloading data with pydrive! In the meantime I met the following error: 

```
Authentication successful.
authorized access to google drive API!
Parsing JSON metadata...
Downloading 70001 files...
Traceback (most recent call last):
  File "download_ffhq_aging.py", line 384, in <module>
    run_cmdline(sys.argv)
  File "download_ffhq_aging.py", line 379, in run_cmdline
    run(**vars(args))
  File "download_ffhq_aging.py", line 357, in run
    download_files(specs, dst_dir, output_size, check_invalid_images, drive=drive, **download_kwargs)
  File "download_ffhq_aging.py", line 162, in download_files
    missing_specs = [spec for spec in file_specs if spec['file_path'] not in done_specs]
  File "download_ffhq_aging.py", line 162, in <listcomp>
    missing_specs = [spec for spec in file_specs if spec['file_path'] not in done_specs]
NameError: free variable 'done_specs' referenced before assignment in enclosing scope
```

Any ideas what could go wrong here? I made no changes to the code

p.s. I tried to add `--check_invalid_images` but the same error happens

**Comment by royorel** [comment]
oops... I added some functionality earlier today and forgot to cover for this case and thus the error was created. This was fixed in commit aa13d7a.

Sorry about that

**Comment by rakhimovv** [comment]
Wow, thanks for such a quick response and solution. Great work!

**Comment by royorel** [comment]
No problem, let me know if there are still problems with this issue

**Comment by royorel** [comment]
@rakhimovv I Found another bug in the code and fixed it, so you should pull again to commit e10e2de.

Sorry about the mess, it's past midnight here and my brain is not fully functioning :)
